src/subgraph/accounts/subgraphQueries.ts

```diff
@@ -104,6 +104,14 @@ export const fetchTopAccountsByReferralFees = (count: number = 20, skip: number
     unclaimedReferralRewardsWeth
   }
 }`;
+export const fetchAccountByWalletAddress = (walletAddress: string) =>
```

Comment:
Wallet and account are both different terms in the subgraph. Please check the logic and avoid mixing up the terms. Use `Wallet` for a user address and use `snxAccount` for Synthetix account ids